src/main/java/org/gridsuite/modification/server/repositories/NetworkModificationRepository.java

```diff
@@ -64,24 +64,47 @@ public void saveModifications(UUID groupUuid, List<? extends ModificationEntity>
     }
 
     @Transactional // To have all move in the same transaction (atomic)
-    public void moveModifications(UUID groupUuid, List<UUID> modifications, UUID before) {
```

Comment:
I know it was before your commits, but is it possible to rename "before"  in something more explicit ? I didn't understand what it was before reading the whole method.

Reply:
fixed